db/migrate/20230404210918_create_like.rb

```diff
@@ -0,0 +1,10 @@
+class CreateLike < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :user, null: false, foreign_key: true
```

Comment:
- [ ] Same here, Kindly edit this reference so the foreign key supposes to be `author_id` but what you did here will make Active Record create it as `user_id`

Reply:
<img width="500" alt="Captura de pantalla 2023-04-04 a la(s) 16 48 52" src="https://user-images.githubusercontent.com/99061965/229940719-bf90c146-7194-4852-9c93-40482be8b9a8.png">